docker/linksets/asgs2geofab/docker-compose.base.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.4"
+
+services:
+  linksets:
+    build:
+      context: .
+      target: python-specific
+    env_file:
+      - ../../../common/common.env
+      - .env
```

Comment:
Not sure if .env files are the way to go here, or simply put ENV values directly into the docker-compose.yaml files? Not a problem, just a thought.